controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -102,6 +105,23 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
         const orgWebhooks = new OrganizationWebhookService(opts.db, authContext.organizationId, opts.logger);
         const fedGraphRepo = new FederatedGraphRepository(opts.db, authContext.organizationId);
         const subgraphRepo = new SubgraphRepository(opts.db, authContext.organizationId);
+        const userRepo = new UserRepository(opts.db);
+
+        const hasAccess = await userRepo.checkUserAccess({
+          userID: authContext.userId,
+          organizationID: authContext.organizationId,
+          rolesToBe: ['admin', 'member'],
```

Comment:
Is it just me or is it a bit confusing that a member role gives you write access? I'd like to propose a different scheme: This is easier to communicate. WDYT?

- Admin - Can create and manage all services.
- Developer - Write and Read access to all objects in the organization
- Read-only - As the name suggests. Only read-only access to all objects in the organization

See https://docs.aiven.io/docs/platform/reference/project-member-privileges

Reply:
Can we go with admin, developer and viewer?

Reply:
Sure

Reply:
Done